2020-11-26 12:22:14.375 +00:00 [INF] Starting up
2020-11-26 12:22:15.309 +00:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-26 12:22:15.311 +00:00 [INF] Hosting environment: Development
2020-11-26 12:22:15.312 +00:00 [INF] Content root path: C:\Source\store-api\store-api
2020-11-26 12:22:15.322 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/  
2020-11-26 12:22:15.354 +00:00 [INF] Request finished in 34.2824ms 404 
2020-11-26 12:25:42.279 +00:00 [INF] Starting up
2020-11-26 12:25:43.075 +00:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-26 12:25:43.077 +00:00 [INF] Hosting environment: Development
2020-11-26 12:25:43.079 +00:00 [INF] Content root path: C:\Source\store-api\store-api
2020-11-26 12:25:43.088 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/  
2020-11-26 12:25:43.122 +00:00 [INF] Request finished in 36.164ms 404 
2020-11-26 12:25:55.440 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger  
2020-11-26 12:25:55.442 +00:00 [INF] Request finished in 2.4143ms 301 
2020-11-26 12:25:55.445 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger/index.html  
2020-11-26 12:25:55.467 +00:00 [INF] Request finished in 22.2585ms 200 text/html;charset=utf-8
2020-11-26 12:25:55.492 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger/swagger-ui.css  
2020-11-26 12:25:55.492 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger/swagger-ui-bundle.js  
2020-11-26 12:25:55.492 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger/swagger-ui-standalone-preset.js  
2020-11-26 12:25:55.522 +00:00 [INF] Sending file. Request path: '/swagger-ui-standalone-preset.js'. Physical path: 'N/A'
2020-11-26 12:25:55.522 +00:00 [INF] Sending file. Request path: '/swagger-ui.css'. Physical path: 'N/A'
2020-11-26 12:25:55.523 +00:00 [INF] Request finished in 31.1786ms 200 text/css
2020-11-26 12:25:55.523 +00:00 [INF] Request finished in 31.1822ms 200 application/javascript
2020-11-26 12:25:55.535 +00:00 [INF] Sending file. Request path: '/swagger-ui-bundle.js'. Physical path: 'N/A'
2020-11-26 12:25:55.535 +00:00 [INF] Request finished in 42.4611ms 200 application/javascript
2020-11-26 12:25:55.652 +00:00 [INF] Request starting HTTP/2.0 GET https://localhost:44307/swagger/v1/swagger.json  
2020-11-26 12:25:56.028 +00:00 [INF] Request finished in 375.9738ms 200 application/json; charset=utf-8
2020-11-26 12:26:00.904 +00:00 [INF] Request starting HTTP/2.0 POST https://localhost:44307/session/basket application/json 21
2020-11-26 12:26:00.910 +00:00 [INF] CORS policy execution failed.
2020-11-26 12:26:00.912 +00:00 [INF] Request origin https://localhost:44307 does not have permission to access the resource.
2020-11-26 12:26:00.913 +00:00 [INF] Executing endpoint 'store_api.Controllers.SessionController.GetCurrentBasket (store-api)'
2020-11-26 12:26:00.934 +00:00 [INF] Route matched with {action = "GetCurrentBasket", controller = "Session"}. Executing controller action with signature System.Threading.Tasks.Task`1[store_api.Controllers.Basket] GetCurrentBasket(store_api.Objects.AuthedRequest) on controller store_api.Controllers.SessionController (store-api).
2020-11-26 12:26:15.388 +00:00 [ERR] Exception getting categories
System.NotImplementedException: The method or operation is not implemented.
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 33
2020-11-26 12:26:15.465 +00:00 [INF] Executed action store_api.Controllers.SessionController.GetCurrentBasket (store-api) in 14527.4665ms
2020-11-26 12:26:15.466 +00:00 [INF] Executed endpoint 'store_api.Controllers.SessionController.GetCurrentBasket (store-api)'
2020-11-26 12:26:15.467 +00:00 [ERR] An unhandled exception has occurred while executing the request.
System.NotImplementedException: The method or operation is not implemented.
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-26 12:26:15.476 +00:00 [INF] Request finished in 14572.8589ms 500 text/plain
2020-11-26 12:28:34.904 +00:00 [INF] Starting up
2020-11-26 12:29:28.691 +00:00 [INF] Starting Application
2020-11-26 13:26:22.865 +00:00 [INF] Starting Application
2020-11-26 13:29:05.763 +00:00 [INF] Starting Application
2020-11-26 13:29:36.735 +00:00 [INF] Starting Application
2020-11-26 13:51:32.830 +00:00 [INF] Starting Application
2020-11-26 13:51:49.020 +00:00 [ERR] store_api.CloudDatastore.DAL.Repositories.SessionRepository.Get experienced a Grpc.Core.RpcException
Grpc.Core.RpcException: Status(StatusCode="FailedPrecondition", Detail="no matching index found. recommended index is:
- kind: Baskets
  properties:
  - name: Historic
  - name: UserUid
", DebugException="Grpc.Core.Internal.CoreErrorDetailException: {"created":"@1606398708.982000000","description":"Error received from peer ipv4:216.58.204.10:443","file":"T:\src\github\grpc\workspace_csharp_ext_windows_x64\src\core\lib\surface\call.cc","file_line":1062,"grpc_message":"no matching index found. recommended index is:\n- kind: Baskets\n  properties:\n  - name: Historic\n  - name: UserUid\n","grpc_status":9}")
   at Google.Api.Gax.Grpc.ApiCallRetryExtensions.<>c__DisplayClass0_0`2.<<WithRetry>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Google.Cloud.Datastore.V1.QueryStreamer.AsyncQueryEnumerable.AsyncQueryEnumerator.MoveNextAsync()
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at Google.Cloud.Datastore.V1.AsyncLazyDatastoreQuery.GetAllResultsAsync()
   at store_api.CloudDatastore.DAL.Repository.Get(DbCollections kind, Filter filter) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repository.cs:line 34
2020-11-26 13:51:49.158 +00:00 [ERR] Exception getting categories
System.Exception: store_api.CloudDatastore.DAL.Repositories.SessionRepository.Get experienced a Grpc.Core.RpcException
 ---> Grpc.Core.RpcException: Status(StatusCode="FailedPrecondition", Detail="no matching index found. recommended index is:
- kind: Baskets
  properties:
  - name: Historic
  - name: UserUid
", DebugException="Grpc.Core.Internal.CoreErrorDetailException: {"created":"@1606398708.982000000","description":"Error received from peer ipv4:216.58.204.10:443","file":"T:\src\github\grpc\workspace_csharp_ext_windows_x64\src\core\lib\surface\call.cc","file_line":1062,"grpc_message":"no matching index found. recommended index is:\n- kind: Baskets\n  properties:\n  - name: Historic\n  - name: UserUid\n","grpc_status":9}")
   at Google.Api.Gax.Grpc.ApiCallRetryExtensions.<>c__DisplayClass0_0`2.<<WithRetry>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Google.Cloud.Datastore.V1.QueryStreamer.AsyncQueryEnumerable.AsyncQueryEnumerator.MoveNextAsync()
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at Google.Cloud.Datastore.V1.AsyncLazyDatastoreQuery.GetAllResultsAsync()
   at store_api.CloudDatastore.DAL.Repository.Get(DbCollections kind, Filter filter) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repository.cs:line 34
   --- End of inner exception stack trace ---
   at store_api.CloudDatastore.DAL.Repository.Get(DbCollections kind, Filter filter) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repository.cs:line 40
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 27
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 42
2020-11-26 13:51:49.196 +00:00 [ERR] An unhandled exception has occurred while executing the request.
System.Exception: store_api.CloudDatastore.DAL.Repositories.SessionRepository.Get experienced a Grpc.Core.RpcException
 ---> Grpc.Core.RpcException: Status(StatusCode="FailedPrecondition", Detail="no matching index found. recommended index is:
- kind: Baskets
  properties:
  - name: Historic
  - name: UserUid
", DebugException="Grpc.Core.Internal.CoreErrorDetailException: {"created":"@1606398708.982000000","description":"Error received from peer ipv4:216.58.204.10:443","file":"T:\src\github\grpc\workspace_csharp_ext_windows_x64\src\core\lib\surface\call.cc","file_line":1062,"grpc_message":"no matching index found. recommended index is:\n- kind: Baskets\n  properties:\n  - name: Historic\n  - name: UserUid\n","grpc_status":9}")
   at Google.Api.Gax.Grpc.ApiCallRetryExtensions.<>c__DisplayClass0_0`2.<<WithRetry>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Google.Cloud.Datastore.V1.QueryStreamer.AsyncQueryEnumerable.AsyncQueryEnumerator.MoveNextAsync()
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at System.Linq.AsyncEnumerable.<ToListAsync>g__Core|620_0[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken) in d:\a\1\s\Ix.NET\Source\System.Linq.Async\System\Linq\Operators\ToList.cs:line 27
   at Google.Cloud.Datastore.V1.AsyncLazyDatastoreQuery.GetAllResultsAsync()
   at store_api.CloudDatastore.DAL.Repository.Get(DbCollections kind, Filter filter) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repository.cs:line 34
   --- End of inner exception stack trace ---
   at store_api.CloudDatastore.DAL.Repository.Get(DbCollections kind, Filter filter) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repository.cs:line 40
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 27
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 42
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-26 13:53:51.924 +00:00 [INF] Starting Application
2020-11-26 14:21:02.067 +00:00 [INF] Starting Application
2020-11-26 14:21:55.292 +00:00 [ERR] Exception getting categories
System.NullReferenceException: Object reference not set to an instance of an object.
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.<>c.<GetCurrentBasket>b__2_0(Entity entity) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 35
   at System.Linq.Enumerable.SelectListIterator`2.TryGetFirst(Boolean& found)
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 35
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 32
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 42
2020-11-26 14:21:57.622 +00:00 [ERR] An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.<>c.<GetCurrentBasket>b__2_0(Entity entity) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 35
   at System.Linq.Enumerable.SelectListIterator`2.TryGetFirst(Boolean& found)
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 35
   at store_api.CloudDatastore.DAL.Repositories.SessionRepository.GetCurrentBasket(String requestUid) in C:\Source\store-api\store-api.CloudDatastore.DAL\Repositories\SessionRepository.cs:line 32
   at store_api.Controllers.SessionController.GetCurrentBasket(AuthedRequest token) in C:\Source\store-api\store-api\Controllers\SessionController.cs:line 42
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-26 14:25:17.171 +00:00 [INF] Starting Application
2020-11-26 14:25:48.144 +00:00 [INF] Starting Application
2020-11-26 14:27:11.649 +00:00 [INF] Starting Application
2020-11-26 14:32:13.156 +00:00 [INF] Starting Application
2020-11-26 14:47:41.598 +00:00 [INF] Starting Application
2020-11-26 14:49:06.160 +00:00 [INF] Starting Application
2020-11-26 15:07:46.885 +00:00 [INF] Starting Application
2020-11-26 15:08:09.470 +00:00 [INF] Starting Application
2020-11-26 15:09:14.873 +00:00 [INF] Starting Application
2020-11-26 15:14:09.484 +00:00 [INF] Starting Application
2020-11-26 15:15:35.721 +00:00 [INF] Starting Application
2020-11-26 15:17:15.534 +00:00 [INF] Unauthorised user request token: string
FirebaseAdmin.Auth.FirebaseAuthException: Incorrect number of segments in ID token.
   at FirebaseAdmin.Auth.Jwt.FirebaseTokenVerifier.VerifyTokenAsync(String token, CancellationToken cancellationToken)
   at FirebaseAdmin.Auth.AbstractFirebaseAuth.VerifyIdTokenAsync(String idToken, Boolean checkRevoked, CancellationToken cancellationToken)
   at FirebaseAdmin.Auth.AbstractFirebaseAuth.VerifyIdTokenAsync(String idToken, CancellationToken cancellationToken)
   at FirebaseAdmin.Auth.AbstractFirebaseAuth.VerifyIdTokenAsync(String idToken)
   at store_api.FirebaseAuthHelper.Verify(AuthedRequest token) in C:\Source\store-api\store-api\FirebaseAuthHelper.cs:line 16
2020-11-26 15:26:12.001 +00:00 [INF] Starting Application
